data/dialog phrases.txt

```diff
@@ -139,3 +151,11 @@ phrase "generic aborted pirate cargo mission"
 phrase "generic completed pirate slave mission"
 	word
 		`After landing on <planet>, the pirates move the <bunks> slaves from your ship and pay you <payment>.`
+
+phrase "generic completed emancipation mission"
+	word
+		`Local authorities thank you with joyous words and <payment>. They lead the <bunks> people you helped emancipate to their newfound freedom.`
+
+phrase "generic completed emancipation raid mission"
+	word
+		`Local authorities thank you with joyous words and <payment>. They lead the people you helped emancipate to their newfound freedom.`
```

Comment:
Is there a reason why this can't be condensed into one phrase?

Reply:
Maybe, but I don't think it needs to be. There's five ideas here, and that's a bit much for one sentence. Some go together quite well, but not all of them:
- Authorities thank the player and express joy.
- Authorities pay the player.
- Authorities lead the people you helped away.
- You helped emancipate these people.
- These people are now free.

Reply:
They're separate because the raid escort missions have no `<bunks>` and thus cannot tell you how many people you rescued.

Reply:
I've rewritten the raiding mission completion message. It makes more sense now, given the context. How do you like it now?